config/routes.rb

```diff
@@ -23,6 +23,12 @@
 
   get '/dashboard', to: 'merchants#show'
   get '/dashboard/items', to: 'merchants#items_index'
+  get '/dashboard/items/new', to: 'merchants#new'
+  post '/items', to: 'merchants#create'
+  delete '/dashboard/items', to: 'merchants#destroy', as: 'delete_item'
+  get '/dashboard/items/update', to: 'merchants#edit', as: 'edit_item'
+  patch '/items', to: 'merchants#update'
+  post '/toggle-item-enable', to: 'merchants#toggle_item', as: 'toggle_item'
 
```

Comment:
Wasn't Leah working under an `Dashboard` namespace on #95? If so that would be cleaner than having these hand rolled ones.

Reply:
probs